src/app/components/utility/modal/modal.component.ts

```diff
@@ -242,6 +277,57 @@ export class ModalComponent implements OnInit {
       this.isDisabled = e.target.value === this.user.last_name;
     } else if (e.target.name === 'update_affiliation') {
       this.isDisabled = e.target.value === this.user.affiliation;
+    } else if (e.target.name === 'update_google_scholar_url') {
+      if (this.validProtocol(e.target.value) && this.validURL(e.target.value)) {
+        this.isDisabled = e.target.value === this.user.google_scholar_url;
+
+        this.invalidFields = this.invalidFields.filter(element => element !== 'Google Scholar');
+        this.convertFieldArrayIntoText(this.invalidFields);
+      } else if (e.target.value === '') {
```

Comment:
And can we move this case to the above `if` condition as only `isDisabled` needs to change a/c to this case?

Reply:
We can't. 

Assume the following code:
```javascript
if (e.target.value === '') {
  this.isDisabled = false;

  this.invalidFields = this.invalidFields.filter(element => element !== 'Google Scholar');
  this.convertFieldArrayIntoText(this.invalidFields);
}

if (this.validURL(e.target.value)) {
  this.isDisabled = e.target.value === this.user.google_scholar_url;

  this.invalidFields = this.invalidFields.filter(element => element !== 'Google Scholar');
  this.convertFieldArrayIntoText(this.invalidFields);
} else {
  if (!(this.invalidFields.find(element => element === 'Google Scholar'))) {
    this.invalidFields.push('Google Scholar');
  }
  this.convertFieldArrayIntoText(this.invalidFields);
}
```

When `e.target.value` is equal to `''`, the `this.invalidFields` array will be empty. But then, it would also be perceived as not a valid URL (`this.validURL`), thus it would go into the else statement, and it will be added back to the array. This would not happen if it was an `else if` function. Since the condition has already exist, it would not go into the `else` case.

Reply:
no no, sorry for mis typo, actually I mean to move this inside `if` condition and modify `isDisabled` accordingly.

Reply:
Done, please take a look..

Reply:
Done, please take a look.